docs/src/concepts.md

```diff
@@ -2,16 +2,24 @@
 
 ## Storage of high-level information about a laboratory 
 
-Most pipelines track some information about the lab, including the facilities, experiment rigs, and users. This can help track which users lab members are associated with which projects or who is responsible for equipment.
+Most pipelines track some information about the lab, including the facilities,
+experiment rigs, and users. This can help track which users lab members are associated
+with which projects or who is responsible for equipment.
```

Comment:
```suggestion
Most pipelines track some information about the lab, including the facilities,
experiment rigs, and users. This can help track which lab members are associated
with which projects or who is responsible for equipment.
```